CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ set(VARS_PREFIX "iDynTree")
 
 set(${VARS_PREFIX}_MAJOR_VERSION 0)
 set(${VARS_PREFIX}_MINOR_VERSION 11)
-set(${VARS_PREFIX}_PATCH_VERSION 1)
+set(${VARS_PREFIX}_PATCH_VERSION 2)
```

Comment:
I am afraid I did not document this, but I think we should follow the YARP versioning scheme documented in https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#example , so the devel version should have thier patch version that should start from a big number such as `100`. I should have done this after the 0.11 release, but as you may imagine not being an automatic step I forgot to do it. : )

Reply:
Is ``102`` ok?

Reply:
ok!

Reply:
Done in https://github.com/robotology/idyntree/pull/517/commits/27c1008229f18d1adba528ca4199f9f3cffb6122